src/main/java/it/grid/storm/namespace/StoRIImpl.java

```diff
@@ -564,6 +566,24 @@ private TTURL buildHTTPTURL(Protocol p, Authority authority) {
     return TTURL.makeFromString(sb.toString());
   }
 
+  private TTURL buildDAVTURL(Protocol p, Authority authority) {
```

Comment:
This function seems identical to `buildHTTPTURL`, apart from the logging, which may be parameterized. Cannot we reuse that function? also in the call site